documentation/docs/steps/neoDeploy.md

```diff
@@ -24,75 +24,91 @@ Note that a version is formed by `major.minor.patch`, and a version is compatibl
 | -------------------|-----------|-------------------------------|-------------------------------------------------|
-| `account`          | no        |                               |                                                 |
-| `applicationName`  | yes       |                               |                                                 |
-| `archivePath`      | no        |                               |                                                 |
-| `deployAccount` deprecated, use `account` | no                 |                                                 |
-| `deployHost` deprecated, use `host` | no                       |                                                 |
-| `deployMode`       | yes       | `'mta'`                       |  `'mta'`, `'warParams'`, `'warPropertiesFile'`  |
-| `host`             | no        |                               |                                                 |
-| `neoCredentialsId` | no        | `'CI_CREDENTIALS_ID'`         |                                                 |
+| `script`           | yes       |                               |                                                 |
+| `neo`              | no        |                               |                                                 |
+| `deployMode`       | yes       | `'mta'`                       | `'mta'`, `'warParams'`, `'warPropertiesFile'`   |
 | `neoHome`          | no        |                               |                                                 |
+| `source`           | no        |                               |                                                 |
+| `warAction`        | yes       | `'deploy'`                    | `'deploy'`, `'rolling-update'`                  |
+
+The parameter `neo` is a map which contains the following parameters:
+
+| parameter          | mandatory | default                       | possible values                                 |
+| -------------------|-----------|-------------------------------|-------------------------------------------------|
+| `account`          | yes       |                               |                                                 |
+| `application`      | yes       |                               |                                                 |
+| `credentialsId`    | no        | `'CI_CREDENTIALS_ID'`         |                                                 |
+| `environment`      |           |                               |                                                 |
+| `host`             | yes       |                               |                                                 |
 | `runtime`          | yes       |                               |                                                 |
 | `runtime-version`  | yes       |                               |                                                 |
```

Comment:
Docs should be fixed, if I am not mistaken the parameter is actually called `runtimeVersion`.

Reply:
Thanks. Fixed.